docs/version0.35.md

```diff
@@ -31,6 +31,8 @@ The following new features and notable changes since version 0.33.1 are included
 - [New `-XX:[+|-]ShowNativeStackSymbols` option added](#new--xx-shownativestacksymbols-option-added)
 - [New `user2` event added for the `-Xdump` option](#new-user2-event-added-for-the--xdump-option)
 - [New `-XX:[+|-]PerfTool` option added](#new--xx-perftool-option-added)
+- [`-XX:+EnsureHashed:java/lang/Class,java/lang/Thread` is added in the `options.default` file](#xxensurehashedjavalangclassjavalangthread-is-added-in-the-optionsdefault-file)
```

Comment:
This link doesn't work in the generated doc. The previous three don't work either.